src/main/scala/beam/agentsim/events/handling/BeamEventsWriterParquet.scala

```diff
@@ -84,7 +84,7 @@ class BeamEventsWriterParquet(
       .build()
   }
 
-  override protected def writeEvent(event: Event): Unit = {
```

Comment:
This is due to the warning:

> weaker access privileges in overriding
> def ... (defined in class BeamEventsWriterBase)
>   override should be public

And I wanted to point it out.

This was done in BeamEventsWriterXML also.